R/Design-class.R

```diff
@@ -524,15 +523,3 @@ DADesign <- function(model,
     safetyWindow = safetyWindow
   )
 }
-validObject(DADesign(
-  model = .DALogisticLogNormal(),
-  data = DataDA(doseGrid = 1:2),
-  safetyWindow = .SafetyWindowConst(),
-  nextBest = .NextBestNCRM(),
-  startingDose = 1,
-  cohortSize = CohortSizeConst(3),
-  stopping = StoppingMinCohorts(10),
-  increments = IncrementsDoseLevels(2)
-))
-
-# nolint end
```

Comment:
maybe you need to move the `nolint end` line just above `DADesign` now since that part up there is not yet cleaned